### ISSUE TITLE
0.19.0 release: Error: Cannot find module './minecraft-data/data/common/protocolVersions'

### ISSUE THREAD

**Issue by deathcap** [PrismarineJS/node-minecraft-data#16]
When installing the 0.19.0 release of minecraft-data from from npm, require() fails:

```
tmp $ npm install minecraft-data
minecraft-data@0.19.0 node_modules/minecraft-data
tmp $ node
> require('minecraft-data')
Error: Cannot find module './minecraft-data/data/common/protocolVersions'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/private/tmp/node_modules/minecraft-data/index.js:3:22)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
```

see also https://circleci.com/gh/PrismarineJS/node-minecraft-protocol/371 - observed when updating in https://github.com/PrismarineJS/node-minecraft-protocol/pull/331

using the git version of node-minecraft-data (https://github.com/PrismarineJS/node-minecraft-data/tree/4e7f910e62ea30c641a9df2170275d49bbd1edad) with `npm install` works locally at least on my system, so I think this is an issue with the release only

**Comment by deathcap** [comment]
0.17.0 and earlier (`npm install minecraft-data@0.17; find node_modules`) contain the minecraft-data submodule, but 0.18.0 and 0.19.0 are missing it

**Comment by rom1504** [comment]
My bad. I was cloning the repo to work around https://github.com/npm/npm/issues/5673#issuecomment-170316360 and I forgot to --recursive.
Fixed in 0.19.1 !
